### PR TITLE
Integrate msgpack with structlog.

### PR DESCRIPTION
**Pull request by JTCunning**
@mattrobenolt

### CODE REVIEW

**Comment by mattrobenolt** [comment]
How does this mingle wiht the celery logger?

**Comment by mattrobenolt** [comment]
And will this cover 100% of logging at this point to be msgpack format?

**Comment by JTCunning** [comment]
1. It will still output with the celery prepended `[2016-06-16 22:20:00,238: WARNING/Worker-1]` noise, which is going to be addressed in a different PR.
2. This will cover 100% of logging if we configure the format to be `machine`. Still trying to make it play well with printing one line stack traces.

File: src/sentry/logging/renderers.py [hunk 1]
```diff
@@ -17,6 +17,10 @@ class MessagePackRenderer(object):
     If you are doing bad things and pass in a native object, it won't fail.
     """
     def __call__(self, logger, name, event_dict):
+        for k, v in event_dict.items():
```
Comment (mattrobenolt) [inline src/sentry/logging/renderers.py]: iteritems

File: src/sentry/runner/initializer.py [hunk 2]
```diff
@@ -155,20 +155,34 @@ def configure_structlog():
...
+                    'event',
+                ]
+            )
+        ])
+    elif fmt == LoggingFormat.MACHINE:
+        from sentry.logging.renderers import MessagePackRenderer
+        processors.append(MessagePackRenderer())
+
+    structlog.configure(
+        processors,
```
Comment (mattrobenolt) [inline src/sentry/runner/initializer.py]: Is this wrong?

**Comment by JTCunning** [comment]
Punting on this for now in favor of json. Too many moving parts are in the way.

**Pull request closed without merging**